spring-integration-zeromq/src/main/java/org/springframework/integration/zeromq/outbound/ZeroMqMessageHandler.java

```diff
@@ -244,7 +255,12 @@ protected Mono<Void> handleMessageInternal(Message<?> message) {
 						if (socket.base() instanceof Pub) {
 							String topic = this.topicExpression.getValue(this.evaluationContext, message, String.class);
 							if (topic != null) {
-								msg.wrap(new ZFrame(topic));
+								var frame = new ZFrame(topic);
+								if (wrapTopic) {
+									msg.wrap(frame);
+								} else {
```

Comment:
Our code style is to have `else` on a new line.
Run `gradlew :spring-integration-zeromq:check` locally.